common/lib/xmodule/xmodule/tests/test_library_content.py

```diff
@@ -5,20 +5,34 @@
 Higher-level tests are in `cms/djangoapps/contentstore/tests/test_libraries.py`.
 """
 import ddt
-from xmodule.library_content_module import LibraryVersionReference, ANY_CAPA_TYPE_VALUE, LibraryContentDescriptor
+from mock import patch
+from unittest import TestCase
+from bson.objectid import ObjectId
+
+from opaque_keys.edx.locator import LibraryLocator
+
+from xblock.fragment import Fragment
+from xblock.runtime import Runtime as VanillaRuntime
+
+from xmodule.x_module import AUTHOR_VIEW
+from xmodule.library_content_module import (
+    LibraryVersionReference, LibraryList, ANY_CAPA_TYPE_VALUE, LibraryContentDescriptor
+)
 from xmodule.modulestore.tests.factories import LibraryFactory, CourseFactory, ItemFactory
 from xmodule.modulestore.tests.utils import MixedSplitTestCase
 from xmodule.tests import get_test_system
 from xmodule.validation import StudioValidationMessage
 
 
-@ddt.ddt
-class TestLibraries(MixedSplitTestCase):
+dummy_render = lambda block, _: Fragment(block.data)  # pylint: disable=invalid-name
+
+
+class BaseTestLibraryContainer(MixedSplitTestCase):
```

Comment:
@e-kolpakov

Nit: I would prefer to name this new base class `LibraryContentTest` since I just had to make the same change in  https://github.com/edx/edx-platform/pull/6492/files#diff-6 and some of the new tests don't have much to do with the "container" nature of LibraryContent block.

I would also prefer that in the other new test class below we use the name "LibraryContent" rather than "LibraryContainer" since content is the name of the block that we're testing, and "container" is a different (but related) concept in Studio.